frontend/src/queries/nodes/InsightQuery/utils/filtersToQueryNode.ts

```diff
@@ -106,7 +102,7 @@ export const legacyEntityToNode = (
             timestamp_field: entity.timestamp_field || undefined,
             distinct_id_field: entity.distinct_id_field || undefined,
             table_name: entity.table_name || undefined,
```

Comment:
The properties still get added here for data warehouse nodes, so they should also still work. I haven't checked this though (don't know how I'd need to setup things).